nginx/tile_multiplier.py

```diff
@@ -3,15 +3,24 @@
 import sys
 
 min_zoom, max_zoom = [int(z) for z in sys.argv[1:3]]
+tile_set = set()
+
+def print_once(z, x, y):
+    if min_zoom <= z <= max_zoom:
```

Comment:
If the produced zoom are bad, we need to fix the range in place.

Reply:
Line 9 is intended to allow the `min_zoom`..`max_zoom` range to be lower or higher than the zoom level in the input list. If, for example, the zoom level in the input is 14, it allows the use of `tile_multiplier.py 0 11` or ``tile_multiplier.py 16 19`.

Reply:
yes !